docs/mkdocs.yml

```diff
@@ -760,9 +760,10 @@ nav:
         - guestsettings list: cmd/teams/guestsettings/guestsettings-list.md
         - guestsettings set: cmd/teams/guestsettings/guestsettings-set.md
       - meeting:
+        - meeting attendancereport list: cmd/teams/meeting/meeting-attendancereport-list.md
```

Comment:
I'd say let's leave this on the place where it was (in front of this command). 
Usually we are sorting everything alphabetically in groups. Meaning all CRUD commands come first, after that we list the next group.

Example where you do this as well:
![image](https://user-images.githubusercontent.com/11723921/233734330-dc5a76aa-c560-4e3c-a19c-dcd90fe26bf0.png)

Reply:
✅